plugins/PHP/Plugins/PerRequestPlugins.php

```diff
@@ -66,16 +66,20 @@ private function __construct()
 
         if(isset($_SERVER['REMOTE_ADDR'])) {
             pinpoint_add_clue(PP_REQ_CLIENT, $_SERVER["REMOTE_ADDR"]);
+        } elseif (($hostname = gethostname()) !== false) {
+            pinpoint_add_clue(PP_REQ_CLIENT, $hostname);
```

Comment:
PP_REQ_CLIENT is the client  information, why not use current hostname?

Reply:
I don't get what you mean, do you mean use `HTTP_HOST` ? Or another field ?

Reply:
PP_REQ_CLIENT , use to trace the client information( eg: IP address of client ).  
`hostname`,  It could confuse user.

Reply:
In case of a CLI run the client is the server running the CLI so that why I've set it at the hostname, can also set it has an the server IP or 127.0.0.1, if i can't get the real IP. Would it make more sense ?

Reply:
In my opinion, if this field can't help you, just drop it(not set).

Reply:
OK, I removed it from `PP_REQ_CLIENT` and combine it into `PP_REQ_SERVER`.